website/data/docs-navigation.js

```diff
@@ -144,6 +144,10 @@ export default [
             category: 'conversion',
             content: ['can', 'convert', 'try'],
           },
+          {
+            category: 'variable',
+            content: ['vault'],
+          },
```

Comment:
Variable functions ?

Reply:
Ah because the output of 'vault' is variable.

Reply:
Ah that is a good point. Originally I had it as Vault Functions but wasn't sure if we would want to categorize differently once consul keys and aws secrete manager were added.Maybe vault functions is the way to go because variable functions does seem a bit vague.

Reply:
I think I would leave this in simply functions, because for now there is only one vault function,  also it feels a bit weird to move a function to another place than functions, but we could have a hashitools folder though ? 

Something like : 
* `functions/hashitools/vault`
* `functions/hashitools/consul`

Reply:
Ah right yes !